lsp-completion.el

```diff
@@ -213,16 +213,16 @@ Return `nil' when fails to guess prefix."
 (defun lsp-completion--to-internal (items)
   "Convert ITEMS into internal form."
   (--> items
-       (-map (-lambda ((item &as &CompletionItem
-                             :label
-                             :filter-text?
-                             :_emacsStartPoint start-point
-                             :score?))
-               `(:label ,(or filter-text? label)
-                 :item ,item
-                 :start-point ,start-point
-                 :score ,score?))
-             it)))
+    (-map (-lambda ((item &as &CompletionItem
+                          :label
+                          :filter-text?
+                          :_emacsStartPoint start-point
+                          :score?))
+            `(:label ,(or filter-text? label)
```

Comment:
Can you make this 
```suggestion
            `( :label ,(or filter-text? label)
```
so the subsequent item will align nicely?

Reply:
approve your suggestion.